src/styleSheetSerializer.js

```diff
@@ -101,11 +111,14 @@ const styleSheetSerializer = {
     let classNames = [...getClassNames(nodes)]
     classNames = filterClassNames(classNames, hashes)
 
-    const style = getStyle(classNames)
+    const [
+      style,
+      classNamesToReplace,
+    ] = getStyleAndClassNamesFromSelectorsByHashes(classNames, hashes)
```

Comment:
Would you mind refactoring this and create two separate functions?

Reply:
Not at all. My thinking was to avoid a bit of code and runtime duplication, but I'll follow your suggestion.